docs/learn/storage/block-sync.md

```diff
@@ -55,6 +55,10 @@ If your node mostly serves the latest data but occasionally serves historic data
 
 Chaindata snapshots offer a faster alternative to Full Sync. A snapshot is a compressed archive (e.g. `.tar.gz` file) of a synced node's data directory. Downloading and extracting a snapshot allows a new node to quickly catch up to the blockchain without processing every block individually. See [Use Chaindata Snapshots](../../misc/operation/chaindata-snapshot.md) for more information.
 
+## AWS AMI
+
+AWS AMI provides a convienient way to launch a fully operational Kaia endpoint node. The AMIs are open to public so that anyone can use. We produce AMIs every day, so the chaindata in the latest AMI should be less than one day behind. See [AWS AMI](./aws-ami.md) for more information.
```

Comment:
```suggestion
AWS AMI provides a convenient way to launch a fully operational Kaia endpoint node. The AMIs are open to the public so that anyone can use them. We produce AMIs every day, so the chaindata in the latest AMI should be less than one day behind. AWS AMI provides the fastest way to run a node, even faster than using a chaindata snapshot. See [Use AWS AMI](../../misc/operation/aws-ami.md) for more information.
```
I suggest moving 'aws-ami.md' page to the Node Quick Reference section for consistency with other pages like Upstream EN and Use Chaindata Snapshots. This page is also more of a guide rather than a concept.

Reply:
Thank you, movded 'aws-ami.md' page to the Node Quick Reference section per your suggestion.
However, using AMI may not be the "fastest" way because of the warmup phase. The launch of the new instance is quick, however the instance becomes "usable" after the warmup phase (to my personal experience). So I removed the additional sentence you suggested, instead added "Why use AMI" section in the `aws-ami.md` to cover the advantage of using AMI.

Reply:
Looks like the location of 'aws-ami.md' file remains unchanged (AS-IS: docs/learn/storage, TO-BE: docs/misc/operation). The same goes for image path (AS-IS: static/image/learn, TO-BE: static/image/misc).

Reply:
Sorry, now I moved `aws-ami.md` page.
Also, moved img files. Thanks for pointing out.